### ISSUE TITLE
"intelligent" pinging

### ISSUE THREAD

**Issue by kubenstein** [kubenstein/dynopoker#5]
i think it would not be a bad idea to keep pinging loop on during day and let app fall a sleep during nights.

Also according to new sleeping policy, and while having one app per account, after few days of intelligent sleeping, app can switch to 24/7 pinging mode if it reaches 650h-till-end-of-the-month day.

**Comment by technolingo** [comment]
Yeah. That would make sense if you're building a location-specific app.
